README.md

```diff
@@ -17,7 +10,7 @@ ASK-A-QUESTION
 <a href="#funders-and-partners">Funders and Partners</a>
 </p>
 
-**[Ask-a-question](https://idinsight.github.io/aaq-core/) lets your users ask questions and receive answers from your content repository in their preferred language**
+**[Ask A Question](https://idinsight.github.io/aaq-core/)** is a free and open-source tool created to help non-profit organizations, governments in developing nations, and social sector organizations utilize **Large Language Models** for responding to citizen inquiries in their **native languages**.
```

Comment:
pet peeve of mine: let's sat "use" instead of "utilize".

Reply:
Done!